### PR TITLE
Update Avito instrumentationDynamic docs

### PR DESCRIPTION
**Pull request by sergioboi**
add configuration window images
change to step by step guide

### CODE REVIEW

File: docs/content/docs/test/Run.md [hunk 1]
```diff
@@ -76,28 +76,29 @@ type: docs
...
-  - `performanceNoE2E` -- только компонентные перформанс тесты
-  - `performance` -- только функциональные перформанс тесты
-- Фильтр тестов в формате `[com.test.Foo[.someTest]]` (*Parameters -> Package prefix*, по умолчанию фильтр пустой, не
-  применяется)
-- Количество запусков. Полезно для [отладки нестабильных тестов]({{< ref "/docs/test/TroubleshootingUI.md" >}})
-  (*Parameters -> Retry count*)
-- Версии API (*Parameters -> * Api*)
+1. Перейдите на [конфигурацию `instrumentationDynamic`](http://links.k.avito.ru/tmctAvitoAndroidInstrumentationDynamic)
+2. Нажмите `Run` ![](https://user-images.githubusercontent.com/8560287/89888381-86d8de80-dbd8-11ea-8369-55b517b07b39.png)
+3. В появивщемся окне настройте окружение для запуска
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Run.md]:
```suggestion
3. В появившемся окне настройте окружение для запуска
```

File: docs/content/docs/test/Run.md [hunk 2]
```diff
@@ -76,28 +76,29 @@ type: docs
...
+    ![](https://user-images.githubusercontent.com/8560287/89889948-344cf180-dbdb-11ea-8f02-22a2921e2220.png)
+    ![](https://user-images.githubusercontent.com/8560287/89889966-38790f00-dbdb-11ea-9867-95cb57515607.png)
+        1. `Test URL` - опциональный. API URL, в который ходят тесты
+        2. `env.DYNAMIC_FILTER` - опциональный. Фильтр по типам тестов
+            - `ui` -- компонентные, E2E, скриншотные, интеграционные тесты
+            - `uiNoE2E` -- как `ui`, но без E2E тестов
+            - `regressionNoE2E` -- `uiNoE2E` + ручные тесты
+            - `regression` -- `ui` + ручные тесты
+            - `performanceNoE2E` -- только компонентные перформанс тесты
+            - `performance` -- только функциональные перформанс тесты
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Run.md]:
```suggestion
            - `performance` -- только функциональные performance тесты
```

**Comment by sboishtyan** [comment]
Screenshot 2020-08-11 at 16 51 56

**Review by eugene-krivobokov — APPROVED** [review]

**Comment by sboishtyan** [comment]
Screenshot 2020-08-11 at 14 01 17

**Pull request merged**